modules/acs-central-services-overview.adoc

```diff
@@ -6,15 +6,51 @@
+** *Scanner V4 Matcher*: The Scanner V4 Matcher performs vulnerability matching. If the Central services Scanner V4 Indexer indexed the image, then the matcher fetches the index report from the indexer and matches the report with the vulnerabilities stored in the Scanner V4 database. If a Secured Cluster services Scanner V4 Indexer performed the indexing, then the matcher uses the index report that was sent from that indexer, and then matches against vulnerabilities. The matcher also fetches vulnerability data and updates the Scanner V4 database with the latest vulnerability data. The Scanner V4 Matcher outputs a vulnerability report, which contains the final results of an image.
+* *Scanner V4 DB*: This database stores information for Scanner V4, including all vulnerability data and index reports. For best performance, configure a persistent volume claim (PVC) for Scanner V4 DB.
+* *StackRox Scanner*: The StackRox Scanner was the default scanner in {product-title-short} before Scanner V4 was introduced in release 4.4. The StackRox Scanner originates from a fork of the Clair v2 open source scanner. You must continue using this scanner for RHCOS node scanning and platform scanning.
+* *Scanner-DB*: This database contains data for the StackRox Scanner.
+
+{product-title-short} scanners analyze each image layer to determine the base operating system and identify programming language packages and packages that were installed by the operating system package manager. They match the findings against known vulnerabilities from various vulnerability sources. In addition, the StackRox Scanner identifies vulnerabilities in the node's operating system and platform. These capabilities are planned for Scanner V4 in a future release.
+
+[id="vulnerability-sources_{context}"]
+== Vulnerability sources
+
+{product-title-short} uses the following vulnerability sources:
+
+* link:https://secdb.alpinelinux.org/[Alpine Security Database]
+* Data tracked in link:https://alas.aws.amazon.com/index.html[Amazon Linux Security Center]
+* link:https://security-tracker.debian.org/tracker/data/json[Debian Security Tracker]
+* link:https://linux.oracle.com/security/oval[Oracle OVAL]
+* link:https://packages.vmware.com/photon/photon_oval_definitions/[Photon OVAL]
+* link:https://access.redhat.com/security/data/oval/v2/[Red Hat OVAL]
+* link:https://access.redhat.com/security/data/metrics/cvemap.xml[Red Hat CVE Map]: This is used for images which appear in the link:https://catalog.redhat.com/software/containers/explore[Red Hat Container Catalog].
+* link:https://support.novell.com/security/oval/[SUSE OVAL]
+* link:https://security-metadata.canonical.com/oval/[Ubuntu OVAL]
+* link:https://osv.dev/[OSV]: This is used for language-related vulnerabilities (Go, Java, Node.js (JavaScript), Python, and Ruby). This source provides GitHub Advisory (GHSA) IDs rather than CVE numbers for vulnerabilities.
```

Comment:
> This source provides GitHub Advisory (GHSA) IDs rather than CVE numbers for vulnerabilities.

It **may** provide GHSAs instead of CVEs

Reply:
Changed to "might" to indicate "possibility" per IBM Style